js_modules/dagster-ui/packages/ui-core/src/graph/CustomKindIcons.tsx

```diff
@@ -0,0 +1,115 @@
+import {Box, Button, Icon, TextInput} from '@dagster-io/ui-components';
+import * as React from 'react';
+
+import {useStateWithStorage} from '../hooks/useStateWithStorage';
```

Comment:
Using local storage for this will likely be problematic in that it is user specific, and won't be visible to team mates. Additionally: Users will lose their configurations when:

    - Switching browsers or devices
    - Clearing cache/local storage
    - Using incognito/private mode
    - Accessing Dagster from a different computer